luigi_pipeline/lib/model/seqr_mt_schema.py

```diff
@@ -344,7 +342,9 @@ def elasticsearch_row(ds):
         table = ds.drop('vep').flatten()
         # When flattening, the table is unkeyed, which causes problems because our locus and alleles should not
         # be normal fields. We can also re-key, but I believe this is computational?
-        table = table.drop(table.locus, table.alleles)
-
+        #
+        # PS: locus and alleles used to be the only possible row keys... however we're extending to drop any row key!
+        row_key = table.row_key
+        table = table.drop(*row_key)
```

Comment:
The keys here are `row_key=['variant_name', 'svtype']`, which is set explicitly during data load on [this step](https://github.com/broadinstitute/seqr-loading-pipelines/pull/398/files#diff-e8711afd284aabadd3dcdc1e1fbbfc1c96dc461aff255765515602a0515529f3R41.) 

We won't actually need them because we've re-annotated to `variantId` and `svType` in [this block](https://github.com/broadinstitute/seqr-loading-pipelines/pull/398/files#diff-d6fdf9e5cbf606552685459442b0989ded7b0f8e4ad7421ef720e73ff69aeca5R60-R69)

Reply:
Gotcha. There is a comment in the function docstring about the dropping locus and alleles key. It's better to update them.

Reply:
Agreed!